### ISSUE TITLE
Which files do I need to "move" a libation instance to a new machine

### ISSUE THREAD

**Issue by CharlieRussel** [rmcrackan/Libation#496]
I'm building a new machine and I would prefer NOT to have to re-do all the settings and login information from my old Libation setup. Which files do I need to copy over from the old machine? (Note the old machine was running a version that was only a couple of minor updates back from the current 9.3.1)

Thanks yet again for a great program!

**Comment by rmcrackan** [comment]
Short version: in Libation's folder, appsettings.json points to the directory where all the other files live. Note the escaped backslashes. Settings.json has a few more important paths in it. FileLocations.json has the paths to where each book was originally downloaded.

None of these 3 files contains sensitive information so if you want to email them to me and tell me your new directories, I could fix them up pretty fast for you.

**Comment by CharlieRussel** [comment]
No need, though the offer is appreciated. I grabbed the Libation folder from another machine and dumped into the new machine. Then I opened up FileLocations.json with VS Code and saw where all the files were located. Which wouldn't work on this new machine, so I created a symbolic link from where Libation thought they should be to where they actually reside. All is well now. (The other two json file pointers were actually pointing to the same locations on the new machine because I am a creature of habit.) The simplest fix was the symbolic link because my D: drive on the new machine is too small for this many audio books. and even with global search and replace, I wasn't keen on replacing all the `"\\D:\\AudibleBooks" with "\\F:\\AudibleBooks"`.
